server/auth/types/jwt/jwt_auth.test.ts

```diff
@@ -34,14 +34,14 @@ describe('test jwt auth library', () => {
     );
   }
 
-  test('test getTokenFromUrlParam', () => {
+  test('test getTokenFromUrlParam', async () => {
     const config = {
       jwt: {
         header: 'Authorization',
         url_param: 'authorization',
       },
     };
-    const auth = getTestJWTAuthenticationHandlerWithConfig(config);
+    const auth = await getTestJWTAuthenticationHandlerWithConfig(config);
```

Comment:
why do you need to have await on both L44 and L26?

Reply:
`getTestJWTAuthenticationHandlerWithConfig `need to call async function: `getAuthenticationHandler.` If the auth handler is not settled, we get the following error messages:

    TypeError: auth.getTokenFromUrlParam is not a function

      50 |
      51 |     const expectedToken = 'testtoken';
    > 52 |     const token = auth.getTokenFromUrlParam(request);
         |                        ^
      53 |     expect(token).toEqual(expectedToken);
      54 |   });
      55 |`

Reply:
Addressed.